<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="wdg_info" native="true">
     <widget class="QLabel" name="lb_tx">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>tx/12.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>这个人很懒，没留下什么</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_name">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>昵称:无名</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_friendlist">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>好友</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_ID">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID:123456789</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdg_choose" native="true">
     <widget class="QPushButton" name="pb_six">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>93</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>六人场</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_seven">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>93</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>七人场</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_nine">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>93</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>九人场</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdg_behavior" native="true">
     <widget class="QPushButton" name="pb_create_room">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>创建房间</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_search_room">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>查询房间</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_tutorial">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>教程</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
